de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,111 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: nagstamon
+rename-desktop-file: nagstamon.desktop
+rename-icon: nagstamon
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --filesystem=xdg-run/keyring
```

Comment:
Is xdg-run/keyring really needed?

Reply:
Yes, to access the Nagstamon secrets in the system key vault like on macOS and Windows.

Reply:
That should work with `org.freedesktop.Secrets` only? Does it use the secrets portal?

Reply:
Without thos permission I get this error:

```
  File "/app/lib/python3.11/site-packages/Nagstamon/Config.py", line 457, in _LoadServersMultipleConfig
    password = keyring.get_password('Nagstamon', '@'.join((servers[server].username,
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/keyring/core.py", line 56, in get_password
    return get_keyring().get_password(service_name, username)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/keyring/backends/fail.py", line 28, in get_password
    raise NoKeyringError(msg)
keyring.errors.NoKeyringError: No recommended backend was available. Install a recommended 3rd party backend package; or, install the keyrings.alt package if you want to use the non-recommended backends. See https://pypi.org/project/keyring for details.
```

Reply:
OK